sangwoo/models/postDao.js

```diff
@@ -0,0 +1,112 @@
+const { dataSource } = require("./dataSource");
+
+const createPost = async (title, content, userId, imageUrl) => {
+  try {
+    return await dataSource.query(
+      `
+        INSERT INTO posts (
+          title, 
+          content,  
+          user_id,
+          image_url
+        ) VALUES 
+        (?, ?, ?, ?);
+        `,
+      [title, content, userId, imageUrl]
+    );
+  } catch (err) {
+    const error = new Error("INVALID_DATA_INPUT");
+    error.statusCode = 500;
```

Comment:
상태 코드 500은 백엔드가 에러처리를 안 했다고 할 수 있습니다. 따라서 500 대신에 400을 사용해 주세요.
(전체적으로 Dao 파일들 확인하고 수정해 주세요.)

```suggestion
    error.statusCode = 400;
```